src/app/components/chat/window/send/send.vue

```diff
@@ -66,6 +56,15 @@ textarea
 		position: absolute
 		top: 10px
 		right: 6px
+
+.content-editor-form-control
```

Comment:
For component styling, prefix with a `-` so that we don't conflict with any global rules, and you can keep the class names short: `.-editor` for example.